frontend/src/components/atoms/OntologyTable.tsx

```diff
@@ -34,16 +34,13 @@ const renderOntology = (
 
 const OntologyTable: React.FC = () => {
   const [selectedNode, setSelectedNode] = useState(initialNode);
-  const [ontologies, setOntologies] = useState([]);
+  const [ontologies, setOntologies] = useState<Array<Ontology>>([]);
+
   const clickNode = async (node: Node) => {
     setSelectedNode(node);
-
-    try {
-      const newOntologies = await getRelations(node.id);
-      setOntologies(newOntologies);
-    } catch (e) {
-      console.log(e);
-    }
+    const newOntologies = await getRelations(node.id);
+    if (newOntologies.length === 0) return;
```

Comment:
I don't think OntologyTable is necessary anymore and should be yeeted. If it's not going to be yeeted we should dispatch an error here.